internal/security/proxy/certs.go

```diff
@@ -58,9 +59,15 @@ func (cs certificate) Load() (*CertPair, error) {
 	if err != nil {
 		return nil, err
 	}
-	cp, err := cs.retrieve(t)
-	if err != nil {
-		return nil, err
+	var cp *CertPair
+	for {
```

Comment:
as mentioned in https://github.com/edgexfoundry/edgex-go/pull/2086/files#r343421187, I think this is the wrong thing to do for master. Can't we use go-mod-secrets in security-proxy-setup? If so then I think we should fix this the _right_ way for master/geneva. 

If this requires changes to go-mod-secrets I'd rather see those in master/geneva than have this on master.

Reply:
> Can't we use go-mod-secrets in security-proxy-setup?

Yes.  But by delaying synchronizing the branches we risk introducing a regression in geneva.

> If so then I think we should fix this the right way for master/geneva.

By vetoing the fix you are essentially signing up to do so.

Reply:
> But by delaying synchronizing the branches we risk introducing a regression in geneva.

Well it's not necessarily a regression since it's always been broken for Geneva... it never worked to begin with so by not merging this code we're not making Geneva any worse than it would have been anyways...

> By vetoing the fix you are essentially signing up to do so.

I disagree